go.mod

```diff
@@ -4,6 +4,7 @@ go 1.16
 
 require (
 	github.com/Azure/go-autorest/autorest/to v0.4.0
+	github.com/giantswarm/apiextensions v0.4.20
```

Comment:
why do we need such an old version of apiexternsions?

Reply:
yeah noticed that too, just wrong package imports